includes/MediaWiki/ReconciliationService.php

```diff
@@ -105,7 +106,9 @@ public function getOrCreateItemByStatementUrl(
 
 		// If we have more than one item matches, something is wrong and we can't edit
 		if ( count( $itemsThatReferenceTheUrlInCorrectStatement ) > 1 ) {
-			die( 'Matched multiple Items during reconciliation :(' );
+			throw new ReconciliationException(
```

Comment:
This exception should probably be caught in `EditEndpoint` (and rethrown as a `LocalizedHttpException` with the same message).

Reply:
This was in fact in EditEndPoint before. And was moved to this file as I was working on this commit.

Reply:
No, I’m saying EditEndpoint should have something like

```php
try {
	// ...
} catch ( ReconciliationException $e ) {
	throw new LocalizedHttpException( $e->getMessageValue(), 400 );
}
```

Reply:
Ah, gotcha. Thanks for the example

Reply:
ok :)